ando/checker.py

```diff
@@ -20,6 +20,24 @@
 
 dir_rules = os.path.join(os.path.dirname(__file__)) + 'rules/'
 def main():
+    """
+     
+    Main file of the validator. uses other class methods for checking
+    different aspects of the directory path.
```

Comment:
Your sentence "uses other class methods for checking different aspects of the directory path." is not very clear, I don't understand it...

Reply:
Yes I think it's an old phrase from an old version that not relevant anymore.

Reply:
Deleted